R/derive_vars_query.R

```diff
@@ -61,14 +61,37 @@
 #'   7, "Alveolar proteinosis", NA_character_, NA_integer_
 #' )
 #' derive_vars_query(adae, queries)
-derive_vars_query <- function(dataset, dataset_queries) {
+derive_vars_query <- function(dataset, dataset_queries) { # nolint: cyclocomp_linter
   assert_data_frame(dataset_queries)
   assert_valid_queries(dataset_queries, queries_name = deparse(substitute(dataset_queries)))
```

Comment:
If we move this after the new checks, we could check the expected variables for `dataset_queries` in one go.